cmd/erasure-multipart.go

```diff
@@ -283,6 +283,17 @@ func (er erasureObjects) newMultipartUpload(ctx context.Context, bucket string,
 		parityBlocks = er.defaultParityCount
 	}
 
+	ecOrg := parityBlocks
+	for _, disk := range onlineDisks {
+		di, err := disk.DiskInfo(ctx)
```

Comment:
```suggestion
		di, err := disk.DiskInfo(ctx)
```

disk can be nil so that protection is needed..